src/main/scala/utils/ErgoNamesUtils.scala

```diff
@@ -76,8 +76,8 @@ object ErgoNamesUtils {
     matches.get(0)
   }
 
-  def buildBoxWithTokenToMint(ctx: BlockchainContext, networkType: NetworkType, value: Long, mintRequestBox: InputBox, tokenDescription: String): OutBox = {
-    val R5_tokenNameBytes = mintRequestBox.getRegisters.get(1).getValue.asInstanceOf[CollOverArray[Byte]].toArray
+  def issuanceBoxArgs(ctx:BlockchainContext, networkType: NetworkType, value: Long, mintRequestBox: InputBox, tokenDescription: String) = {
```

Comment:
does this method need a `BlockchainContext`?

Reply:
👍  removed this from the parameters.